cli/flags.rs

```diff
@@ -803,6 +807,14 @@ Ignore formatting a file by adding an ignore comment at the top of the file:
         .help("Check if the source files are formatted")
         .takes_value(false),
     )
+    .arg(
+      Arg::with_name("ext")
+        .long("ext")
+        .help("Set standard input (stdin) content type")
```

Comment:
I think `--kind` would make more sense. It's more about the type of file rather than the extension...

Reply:
`--kind` sounds more appropriate to me. @bartlomieju wdyt?

Reply:
Something to consider, in dprint I settled on:

```ts
dprint fmt --stdin <file-path/file-name>
```

The reason is that if you provide a full file path it will format according to the extension and apply the inclusion/exclusion rules specified in the dprint config file (so if the file is excluded, then it will stdout the inputted text). If you provide just the file name, then it will always format according to the extension.

This would be useful if in deno you ever had some configuration to ignore files then someone wouldn't have to re-implement the logic and could provide a full file path instead.

Reply:
> This would be useful if in deno you ever had some configuration to ignore files then someone wouldn't have to re-implement the logic and could provide a full file path instead.

Still debating about config files, but I'm not entirely sold on the idea of passing filename if reading from stdin, effectively users would have to put dummy file.

> I think --kind would make more sense. It's more about the type of file rather than the extension...

Just discussed offline, let's go with `--ext`

Reply:
@bartlomieju could be:

```ts
deno fmt --stdin ts
deno fmt --stdin js
deno fmt --stdin tsx

# then if config files are ever added in the future and you want the exclusion/inclusion rules
deno fmt --stdin /home/david/dev/my-project/file.ts
```